doc/developer-notes.md

```diff
@@ -189,9 +191,18 @@ Not OK (used plenty in the current source, but not picked up):
 A full list of comment syntaxes picked up by Doxygen can be found at http://www.doxygen.nl/manual/docblocks.html,
 but the above styles are favored.
 
+Other recommendations:
+
+- Avoid redundant type and in/out information in function descriptions
+- Use backticks (&#96;&#96;) to refer to `argument`s in function descriptions
+- Backticks aren't required when referring to a Function() that Doxygen already knows about; it will build a hyperlink automatically
+- Avoid linking to external documentation; links can break
+- `STRIP_CODE_COMMENTS = YES` in Doxyfile.in, so if you want a standalone comment to be preserved in the Doxygen-generated source code, it must either not conform to Javadoc or be qualified (e.g. begin with `\file`, etc).
```

Comment:
End all or none of these item sentences with a period/full stop, e.g. "."; in this document the other item lists use periods/full stops.

Reply:
Wrapping lines at 80 or 120 characters for these changes could be nice for source code readability on smaller screens.